music/src/core/sequences.ts

```diff
@@ -424,3 +424,95 @@ export function mergeInstruments(ns: INoteSequence) {
 
   return result;
 }
+
+/**
+ * Splits an unquantized `NoteSequence` into chunks.
+ *
+ * @param ns The `NoteSequence` to split.
+ * @param chunkSize The number of steps to split the note sequence in.
+ * For example, if you want to split the sequence into 2 bar chunks, then
+ * if the sequence has 4 steps/quarter, that will be 32 steps for each 2 bars.
+ *
+ * @returns An array of `NoteSequences` each of which are at most `chunkSize`
+ * steps.
+ */
+export function split(seq: INoteSequence, chunkSize = 32): NoteSequence[] {
+  assertIsQuantizedSequence(seq);
+
+  // Make a clone so that we don't destroy the input.
+  const ns = clone(seq);
+
+  // Sort notes first.
+  const notesBystartStep =
+      ns.notes.sort((a, b) => a.quantizedStartStep - b.quantizedStartStep);
+
```

Comment:
We should probably explicitly clear all of the absolute times since they will no longer be valid.

Reply:
did below (when creating the new note)